Documentation/GeneralConventions/FileStructure.rst

```diff
@@ -8,219 +8,71 @@ File structure
     :class: compact-list
     :depth: 2
     :local:
 
-
 .. _file-structure-general:
 
 General
 =======
 
 In order for the documentation to be rendered, you need at least
 
-#.  an index file in one of the following locations in weighted order:
-
-    -  :file:`Documentation/Index.rst`
-    -  :file:`Documentation/index.rst`
-    -  :file:`Documentation/Index.md`
-    -  :file:`Documentation/index.md`
-    -  :file:`README.rst`
-    -  :file:`README.md`
-
-#.  and a theme configuration file under
-
-    - :file:`Documentation/Settings.cfg`.
+*   A valid :file:`composer.json`
+*   A documentation entry point either for :ref:`Full documentation <full-documentation>`
+    or for the rendering of a :ref:`single README file <single-file-documentation>`.
 
 Further conventions are:
 
-*  reST files have the extension **.rst**.
-*  Markdown files have the extension **.md**.
-*  Included reST files have the extension **.rst.txt**.
-*  Use **CamelCase** for directory and file names,
-   for example: Documentation/GeneralConventions/FileStructure.rst.
-
-These conventions pave the way for the *full documentation* and the
-*single file documentation* style, the first being preferred for several
-reasons.
-
+*   reST files have the extension :file:`.rst'.
+*   Markdown files have the extension :file:`.md'.
+*   Included reST files have the extension :file:`.rst.txt'.
+*   Use **CamelCase** for directory and file names,
+    for example: :file:`Documentation/GeneralConventions/FileStructure.rst`.
 
 .. index:: Full documentation
 .. _full-documentation:
 
 Full documentation
 ==================
 
-This is the recommended setup and is commonly used in the official TYPO3 manuals
-and extension documentation.
-
-This structure splits the documentation for the VCS host (README.rst) and the
-Sphinx theme (Documentation/) and allows full use of a continuously expanded set
-of :ref:`custom content elements <rest-reference>`, like UML diagrams, which are
-only supported by the theme. This structure allows to add multiple pages to
-the documentation and build a full page tree.
-
-The Settings.cfg configuration file allows you to set theme variables, i.e. the
-project title, release version and the like.
+To render a complete documentation manual you need a folder called
+:file:`Documentation` with at least an entry point reStrutured Text file called
+:file:`Documentation\Index.rst` and a configuration file called
+:file:`Documentation\guides.xml`. Add more files as needed
 
-Of course, you can also use a README.md instead of a README.rst file as both
-markup languages are supported by the common VCS hosts.
+You can keep :file:`a README.md` or :file:`README.rst` file with basic
+information and a link to the published manual. These files will be commonly
+displayed on GitHub and GitLab.
 
 .. code-block:: none
 
    .
+   ├── composer.json
    ├── README.rst
    └── Documentation
-       ├── genindex.rst
-       ├── Includes.rst.txt
+       ├── About.rst
```

Comment:
Where comes this "About.rst" from?

Reply:
Just to show that you also have normal content here